### PR TITLE
[CDAP-4858] fix namespace delete bug in user store

### PR DESCRIPTION
**Pull request by elfenheart**
- Clean up user store on namespace deletion
- Fix double alert issue
- Fix namespace refresh issue on namespace creation

### CODE REVIEW

File: cdap-ui/app/features/admin/controllers/namespace/settings-ctrl.js [hunk 1]
```diff
@@ -45,10 +45,19 @@ angular.module(PKG.name + '.feature.admin')
 
     };
 
+    /**
```
Comment (anwar6953) [inline cdap-ui/app/features/admin/controllers/namespace/settings-ctrl.js]: If we don't do steps 1,2,3, then the consequence is simply that when the namespace is recreated in the future, the user will then be able to see the templates, drafts, and dashboard, right?
(These 3 things aren't visible in other namespaces, right)?

Comment (elfenheart) [inline cdap-ui/app/features/admin/controllers/namespace/settings-ctrl.js]: yes, that's correct

Comment (anwar6953) [inline cdap-ui/app/features/admin/controllers/namespace/settings-ctrl.js]: One downside I see to this approach is that this requires that the UI manages some aspects of namespace deletion.
For instance, if a user deletes a namespace from the CLI or via `curl`, these things will not get cleaned up, and so they will be visible the next time the namespace is created.
Perhaps a namespaced-storage can be used instead, and the deletion of the namespaced configurations would be managed by the namespace delete action in the backend.
@bdmogal Any suggestions?

Comment (anwar6953) [inline cdap-ui/app/features/admin/controllers/namespace/settings-ctrl.js]: I don't think there is scope for changing this in the backend, so perhaps we can go about the approach in this PR, but file a JIRA to ensure cleanup of those things, even when namespace delete is done outside the UI.

Comment (elfenheart) [inline cdap-ui/app/features/admin/controllers/namespace/settings-ctrl.js]: sounds good. Thanks @anwar6953

**Comment by ajainarayanan** [comment]
:+1: LGTM

The hiding of success of message is not resolved in this PR. We should create a separate PR for that @astewes

**Comment by elfenheart** [comment]
Closing PR. The right fix will be coming from the backend

**Pull request closed without merging**